ui/changes/src/graph.js

```diff
@@ -21,80 +21,36 @@ function generateData(count, yrange) {
   return series;
 }
 
-function rerender(connections, teamComponentMapping) {
+async function generateData() {
+  let map = await getComponentRegressionMap();
+  let return_map = [];
+  Object.keys(map)
```

Comment:
Nit: instead of having a list which you fill up in `.forEach`, you could use `.map`:
```
return Object.keys(map).map((element) => {
    ...
    return values;
});
```

Simple example:
```
return [1, 2, 3, 4].map((element) => element * 2);
```
instead of:
```
let array = [];
[1, 2, 3, 4].forEach((element) => array.push(element * 2));
return array;
```